equinox/_tree.py

```diff
@@ -229,7 +250,11 @@ def _make_replacement(x: _Node) -> Any:
                 "  >>> eqx.tree_at(lambda t: t[0], (None, None, 1), True,\n"
                 "  ...             is_leaf=lambda x: x is None)"
             )
-
+    out = jtu.tree_map(
+        lambda x: x if not isinstance(x, _DistinctTuple) else (),
```

Comment:
Nit: both here and above, can you remove the `not` and flip the branches?

Reply:
done